modules/src/Microsoft.Azure.IIoT.Modules.OpcUa.Publisher/src/Controller/PublisherMethodsController.cs

```diff
@@ -64,10 +64,10 @@ public class PublisherMethodsController : IMethodController {
         /// <summary>
         /// Handler for GetConfiguredEndpoints DM
         /// </summary>
-        public async Task<List<PublishedNodesResponseApiModel>> GetConfiguredEndpointsAsync() {
+        public async Task<List<PublishNodesRequestApiModel>> GetConfiguredEndpointsAsync() {
```

Comment:
Using a request API class for response kind of defeats the purpose of naming it request API model. Either the name of the class should be changed to reflect the entity that it holds and untie it from request/response parts or a new class should be created.

Reply:
Should we rather rename this class then?
as @cristipogacean mentioned in a comment above that we shouldn't be creating a duplicate class for this.

Reply:
@monikavar, I also think that we don't need to introduce duplicates of the classes. So let's rename it to `EndpointApiModel` so that we use it in both requests and responses without introducing any confusion.